src/parser/warrior/arms/modules/talents/AngerManagement.js

```diff
@@ -55,22 +61,20 @@ class AngerManagement extends Analyzer {
   }
 
   get tooltip() {
-    return COOLDOWNS_AFFECTED_BY_ANGER_MANAGEMENT.reduce((a, e) => {
+    return this.cooldownsAffected.reduce((a, e) => {
       return `${a}${SPELLS[e].name}: ${formatDuration(this.effectiveReduction[e] / 1000)} reduction (${formatDuration(this.wastedReduction[e] / 1000)} wasted)<br>`;
     }, '');
   }
 
-  statistic() {
+  subStatistic() {
     return (
-      <StatisticBox
-        icon={<SpellIcon id={SPELLS.ANGER_MANAGEMENT_TALENT.id} />}
+      <StatisticListBoxItem
+        title={<><SpellLink id={SPELLS.ANGER_MANAGEMENT_TALENT.id} /> cooldown reduction</>}
```

Comment:
this is running into the value for me. you should abbreviate "cooldown reduction" somehow (I'd use CDR, personally. the tooltip clarifies what the time means)
![screenshot from 2018-11-04 19-05-14](https://user-images.githubusercontent.com/4909458/47971926-b1d9ea00-e064-11e8-9f7b-1ae096d16b82.png)